database/scripts/schema_converter_migrate.py

```diff
@@ -0,0 +1,197 @@
+"""
+Script to cut over data from the schema converter cache to the upgraded ukrdc 
```

Comment:
Although this is correct in the "arms cache" sense of the word I'd be tempted to use "archive" or "database" or somesuch, as it's not a cache in the computer sense.